### PR TITLE
Newsletter option added, footer modified, cleaned markdown links

### PR DESCRIPTION
**Pull request by ayushjain01**

### CODE REVIEW

**Review by tankala — CHANGES_REQUESTED** [review]
In footer links which are there in header not needed. In footer we need to give option to subscribe to newsletter.

Please add "piptrends home" link in the header

**Comment by ayushjain01** [comment]
Made the required changes

**Review by tankala — CHANGES_REQUESTED** [review]
Need to add widget which will take mail id and subscribe in footer where it mentions pip trends newsletter. This embed script also mentioned in the ticket.

newsletter.md is missing redirect_to

**Comment by ayushjain01** [comment]
unable to change bg colour for substack iframe
newsletter.md has redirect_to

**Review by tankala — CHANGES_REQUESTED** [review]
Decrease the height. unnecessary white space coming.
Mention that it is for pip trends newsletter

**Comment by ayushjain01** [comment]
Unable to override default padding and background colour
image

**Review by tankala — CHANGES_REQUESTED** [review]
Will explain over the call. In footer previous one is good but size need to be decreased. I just wanted to add pip Trends newsletter text that's it.
Current one is bulky.

**Comment by ayushjain01** [comment]
Substack does not have an API, so it's little difficult to modify the signup form.
https://substackapi.com/ is an alternative but requires one additional step to sign up, which can result in lesser conversions. 
Buttondown has an api, but even that requires an email verification to sign up.
I'm trying to build something of our own to make it easier to subscribe on substack itself. 
This is how the footer looks for now-
image

File: _includes/footer.html [hunk 1]
```diff
@@ -2,6 +2,10 @@
 <footer class="footer py-4 js-page-footer">
   <div class="main">
     <center><a href = "../contribute">Click here</a> to contribute an article to learn-pip-trends.com</center>
+    <center>
+      <p>Join our mailing list. Subscribe to pip Trends Weekly Newsletter.</p>
```
Comment (tankala) [inline _includes/footer.html]: Change the text to "Stay Ahead of the Curve: Get the Pip Trends Weekly Digest!"

File: _includes/aside/toc.html [hunk 2]
```diff
@@ -4,5 +4,7 @@
     </div>
     <div class="contribute-box">
         <p><a href="../../contribute" class="contribute-button">Contribute an Article</a></p>
+        <br>
```
Comment (tankala) [inline _includes/aside/toc.html]: Here it takes the user to the newsletter so this button not required. On top anyway we have newsletter option in menu level.

**Review by tankala — CHANGES_REQUESTED** [review]
Please look into comments

**Review by tankala — APPROVED** [review]

**Review by tankala — CHANGES_REQUESTED** [review]
In mobile view subscribe to news it's not looking good check once.
The message and the newsletter should be in the same white box so that people will it is for that.
If possible put the pip Trends part or whole subscribe message in the green color just like our theme.

File: _includes/footer.html [hunk 3]
```diff
@@ -1,7 +1,11 @@
-
-<footer class="footer py-4 js-page-footer">
+<hr>
+<footer class="footer py-4 js-page-footer" >
   <div class="main">
     <center><a href = "../contribute">Click here</a> to contribute an article to learn-pip-trends.com</center>
+    <center>
```
Comment (tankala) [inline _includes/footer.html:4]: Only this part needs to be white. Rest keep as it is because this section and other sections seem like merged.

**Review by tankala — CHANGES_REQUESTED** [review]

**Review by tankala — APPROVED** [review]

**Pull request merged**